src/Rules/Program.hs

```diff
@@ -19,15 +20,16 @@ import Target
 import UserSettings
 import Util
 
--- | Wrapper is an expression depending on the 'FilePath' to the wrapped binary.
-type Wrapper = FilePath -> Expr String
+-- | Wrapper is an expression depending on the 'FilePath' to the
+-- | library path and name of the wrapped binary.
+type Wrapper = FilePath -> FilePath -> Expr String
```

Comment:
Now that `Wrapper` is more complicated, perhaps we should make it a bit more structured? For example:

```haskell
data WrappedBinary = WrappedBinary { binaryLibPath :: FilePath, binaryName :: FilePath }

type Wrapper = WrappedBinary -> Expr String
```
I think that will make code more readable.